charts/seed-bootstrap/charts/loki/values.yaml

```diff
@@ -1,6 +1,7 @@
 global:
   images:
     loki: image-repository:image-tag
+    loki-curator: image-repository:image-tag
```

Comment:
```suggestion
    curator: image-repository:image-tag
```

Reply:
With the code below this image actually is not taken.
New image with key `loki-curator` is included dynamically and is taken in the statefulset.

Here we should decide if we want the image key to be "loki-curator" or just "curator" and to replace it everywhere in the same way